lib/api/Configuration.ts

```diff
@@ -134,11 +140,20 @@ export class Configuration {
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       let configurationDocument: any
       try {
-        configurationDocument = await got(configurationUrl, {
-          username: this.confluenceUser,
-          password: this.confluencePassword,
-          // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        }).json<any>()
+        if (this.confluencePersonalAccessToken !== '') {
+          configurationDocument = await got(configurationUrl, {
```

Comment:
I should've used axios or something that can create a prepared request runner with either basic auth or token auth. Then you wouldn't have to change that much.

Reply:
Hätte hätte Fahrradkette ;-)
See it as room for future improvement.